examples/heatTransfer/read/heatRead_adios2.cpp

```diff
@@ -109,7 +111,7 @@ int main(int argc, char *argv[])
     double *T = new double[vT->m_AvailableSteps * readsize[0] * readsize[1]];
```

Comment:
@williamfgc Yeah, I kind of know this. I am just trying to avoid making too many changes. Because this is not my code, if I make too much modification then it might break something else while fixing the problem I have now. Do you want me to close cancel this pull request? I can use Linux temporarily until someone fixes it.